### DISCUSSION TITLE
i can open webui,but i cant do anything.

### DISCUSSION THREAD

**Discussion by zpskt** [argoproj/argo-workflows#8064] [Q&A]
the web shows err is :
> Failed to load version/info Error: Request has been terminated Possible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unloaded, etc.

**Comment by krzwiatrzyk** [comment]
Same, using Traefik IngressRoute to access dashboard. 

Configuration pretty straightforward, only --auth-mode=server applied

**Reply by krzwiatrzyk** [reply]
Ok, so I guess it may be connected with not trusted certificates by argo? I have changed entrypoint from **websecure** (HTTPS) to **web** (HTTP) and it is okay now
